packages/server/package-lock.json

```diff
@@ -25736,3 +25736,4 @@
 		}
```

Comment:
Remove file

Reply:
If I git rm this file I get a conflict.

I literally went to dev branch and copied in what's in dev as of 3 min ago.

I _think_ this is git getting confused because my branch is older so it sees it as a change?

I'd try rebasing, but every time I rebase I get gnarly package-lock conflicts... I feel like I'm the joplin ourobouros